tests/functional/utils/model_management.py

```diff
@@ -34,6 +36,9 @@ def wait_endpoint_setup(container):
         except Exception as e:
             time.sleep(1)
     print("Logs from container: ", logs)
+    # NOTE(kromanow): this sleep was added to check if occasionall fail on
```

Comment:
this comment sounds like work in process. are the checks concluded.
there is not need to add the note author ;-)

Reply:
I consider this as a good practice to add the note author. I've seen this in many repositories.
About this sleep: this resolved previously occurring problem.
This comment don't necessary state about WIP but about known problem that we're willing to fix in future. Having sleep for a few seconds is workaround, it would be better to implement mechanism ensuring server readiness.
I vote to leave it. Or if not at least create a JIRA task.